reference/config_files/registry.txt.rst

```diff
@@ -55,12 +55,12 @@ Here you have an example of the file *registry.json*:
         ],
         "references":{
             "nanomsg/1.1.2@bincrafters/stable":"bincrafters",
-            "Poco/1.9.0@pocoproject/stable:09378ed7f51185386e9f04b212b79fe2d12d005c":"conan-center",
+            "poco/1.9.4@:09378ed7f51185386e9f04b212b79fe2d12d005c":"conan-center",
```

Comment:
This section is obsolete. The file is now remotes.json, and it doesn't contain references or package_references anymore.

Leave these changes here, and do a PR to fix this. Thanks!

Reply:
understood.

Reply:
I've opened it here: https://github.com/conan-io/docs/issues/1545

Reply:
thanks Javi!